tests/generation/test_token_healing.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a clone of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from parameterized import parameterized
+
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+class TokenHealingTestCase(unittest.TestCase):
```

Comment:
I'm going to ask to move this test class to `tests/generation/test_utils.py`. By default, we only create a new file when there is significant content for the file in question 🤗 

The imports on the file I'm suggesting are correct, so it remove the CI issues!